distribution/pom.xml

```diff
@@ -160,17 +160,32 @@
                         <configuration>
                             <!-- We don't include the v5migrationhelper modules here,
                                  so as not to confuse the documentation with obsolete API -->
+                            <!-- While the generated sources will unlikely contain any public classes for which we
+                                 have to generate any JavaDocs, we still include them for completeness -->
                             <sourcepath>
                                 ${basedir}/../engine/src/main/java;
                                 ${basedir}/../util/common/src/main/java;
                                 ${basedir}/../mapper/pojo-base/src/main/java;
                                 ${basedir}/../mapper/orm/src/main/java;
+                                ${basedir}/../mapper/pojo-standalone/src/main/java;
                                 ${basedir}/../backend/elasticsearch/src/main/java;
                                 ${basedir}/../backend/elasticsearch-aws/src/main/java;
                                 ${basedir}/../backend/lucene/src/main/java;
-                                ${basedir}/../mapper/lucene/src/main/java;
                                 ${basedir}/../mapper/orm-outbox-polling/src/main/java;
                                 ${basedir}/../mapper/orm-jakarta-batch/core/src/main/java;
+                                ${basedir}/../mapper/orm-jakarta-batch/jberet/src/main/java;
+                                ${basedir}/../engine/target/generated-sources/annotations;
+                                ${basedir}/../util/common/target/generated-sources/annotations;
+                                ${basedir}/../mapper/pojo-base/target/generated-sources/annotations;
+                                ${basedir}/../mapper/orm/target/generated-sources/annotations;
+                                ${basedir}/../mapper/pojo-standalone/target/generated-sources/annotations;
+                                ${basedir}/../backend/elasticsearch/target/generated-sources/annotations;
+                                ${basedir}/../backend/elasticsearch-aws/target/generated-sources/annotations;
+                                ${basedir}/../backend/lucene/target/generated-sources/annotations;
+                                ${basedir}/../mapper/orm-outbox-polling/src/main/avro/generated;
+                                ${basedir}/../mapper/orm-outbox-polling/target/generated-sources/annotations;
+                                ${basedir}/../mapper/orm-jakarta-batch/core/target/generated-sources/annotations;
+                                ${basedir}/../mapper/orm-jakarta-batch/jberet/target/generated-sources/annotations;
```

Comment:
Hey @yrodiere 😃 
wanted to check with you if this makes sense to you, or would you say that we shouldn't include the generated sources here?

Reply:
I guess it cannot hurt, since there is not a lot of generated sources and even less API generated sources. Sure, let's do it :)